cmd/hubagent/workload/setup.go

```diff
@@ -215,12 +220,20 @@ func SetupControllers(ctx context.Context, wg *sync.WaitGroup, mgr ctrl.Manager,
 			return err
 		}
 
-		klog.Info("Setting up cluster resource placement eviction controller")
-		if err := (&clusterresourceplacementeviction.Reconciler{
-			Client: mgr.GetClient(),
-		}).SetupWithManager(mgr); err != nil {
-			klog.ErrorS(err, "Unable to set up cluster resource placement eviction controller")
-			return err
+		if opts.EnableEvictionAPIs {
+			for _, gvk := range evictionGVKs {
+				if err = utils.CheckCRDInstalled(discoverClient, gvk); err != nil {
+					klog.ErrorS(err, "unable to find the required CRD", "GVK", gvk)
```

Comment:
```suggestion
					klog.ErrorS(err, "Unable to find the required CRD", "GVK", gvk)
```
log message starts with upper case.